### PR TITLE
translatepy: init at 2.0

### LINKED ISSUE #132079: Packaging Request: translatepy
**Project description**
Translate, transliterate, get the language of texts in no time with the help of multiple APIs!

**Metadata**

* homepage URL: https://github.com/Animenosekai/translate
* source URL: https://github.com/Animenosekai/translate
* license: agpl3
* platforms: linux*

\* It (most likely) runs on many platforms, but I am specifically interested in linux support.

### PR DESCRIPTION
**Pull request by emilytrau**
###### Motivation for this change

Fixes #132079

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pyuseragents/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "Animenosekai";
+    repo = "useragents";
+    rev = "v${version}";
+    sha256 = "D7Qs3vsfkRH2FDkbfakrR+FfWzQFiOCQM7q9AdJavyU=";
+  };
+
+  nativeBuildInputs = [ pytestCheckHook ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyuseragents/default.nix:18]:
```suggestion
  checkInputs = [ pytestCheckHook ];
```

File: pkgs/development/python-modules/translatepy/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  };
+
+  propagatedBuildInputs = [
+    requests
+    beautifulsoup4
+    pyuseragents
+    inquirer
+  ];
+
+  nativeBuildInputs = [ pytestCheckHook ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/translatepy/default.nix:29]:
```suggestion
  checkInputs = [ pytestCheckHook ];
```

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 133457` at 0de7d6d1 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built successfully:</summary>
  <ul>
    <li>python38Packages.pyuseragents</li>
    <li>python38Packages.translatepy</li>
    <li>python39Packages.pyuseragents</li>
    <li>translatepy (python39Packages.translatepy)</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-imports-check-typo.md">python-imports-check-typo</a>

A likely typo in the `pythonImportsTest` argument was found. Did you mean `pythonImportsCheck`?

Near pkgs/development/python-modules/pyuseragents/default.nix:20:3:
```
   |
20 |   pythonImportsTest = [ "pyuseragents" ];
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-imports-check-typo.md">python-imports-check-typo</a>

A likely typo in the `pythonImportsTest` argument was found. Did you mean `pythonImportsCheck`?

Near pkgs/development/python-modules/translatepy/default.nix:34:3:
```
   |
34 |   pythonImportsTest = [ "translatepy" ];
   |   ^
```
</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 133457` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
4 packages built:
</summary>
<ul>
<li>python38Packages.pyuseragents</li>
<li>python38Packages.translatepy</li>
<li>python39Packages.pyuseragents</li>
<li>translatepy (python39Packages.translatepy)</li>
</ul>
</details>

**Pull request merged**